### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by marcusmuller** [An0nimia/deezloader#5]
Is correct?
![image](https://user-images.githubusercontent.com/28604497/199963990-d4b3f592-97d3-4944-8df9-ab4b9ed90242.png)

**Comment by marcusmuller** [comment]
![image](https://user-images.githubusercontent.com/28604497/199970023-f3e8c56c-55df-48a3-8a30-5f300c09471d.png)

[Running] C:\Users\user\AppData\Local\Programs\Python\Python310\python.exe "c:\Users\user\AppData\Local\Programs\Python\Python310\Scripts\deez-dw.py"
usage: deez-dw.py [-h] [-so {dee,spo}] [-l LINK] [-s SONG] [-a ARTIST]
                  [-o OUTPUT]
                  deez_settings
deez-dw.py: error: the following arguments are required: deez_settings

	d8888b. d88888b d88888b d88888D        d8888b. db   d8b   db 
	88  `8D 88'     88'     YP  d8'        88  `8D 88   I8I   88 
	88   88 88ooooo 88ooooo    d8'         88   88 88   I8I   88 
	88   88 88~~~~~ 88~~~~~   d8'   C8888D 88   88 Y8   I8I   88 
	88  .8D 88.     88.      d8' db        88  .8D `8b d8'8b d8' 
	Y8888D' Y88888P Y88888P d88888P        Y8888D'  `8b8' `8d8'


[Done] exited with code=2 in 1.454 seconds

**Comment by marcusmuller** [comment]
*pwd fix for password

**Comment by An0nimia** [comment]
Solved?

**Comment by devinschumacher** [comment]
yeah this doesnt work at all. constant "error with your settings file"
